src/main/java/org/opentripplanner/api/support/ZoneIdFallback.java

```diff
@@ -0,0 +1,39 @@
+package org.opentripplanner.api.support;
```

Comment:
Can you move this here: `org.opentripplanner.framework.time` ? 

Accessing the `org.opentripplanner.api` is not allowed from e.g. domain. This class only have dependencies to Java time, so it is fine to put it into framework time utils.